javascript-data-model/order-history.js

```diff
@@ -0,0 +1,73 @@
+const orderHistory = [
+  {
+    orderPlaced: 'August 4, 2020',
+    total: 34,
+    shipTo: 'JS Masher',
+    orderNumber: '114-3941689-8772232',
+    delivered: 'August 8, 2020',
+    item: [
+      {
+        type: 'book',
+        title: 'JavaScript for impatient programmers',
+        author: 'Rauschmayer, Dr.Axel',
+        returnWindow: 'Sep 7, 2020',
+        totalCost: 31.55,
+      },
+    ],
+  },
+  {
+    orderPlaced: 'July 19, 2020',
+    total: 44.53,
+    shipTo: 'JS Masher',
+    orderNumber: '113-9984268-1280257',
+    delivered: 'July 20, 2020',
+    item: [
+      {
+        type: 'book',
+        title: 'The Timeless Way of Building',
+        author: 'Alexander, Christopher',
+        returnWindow: 'Aug 19, 2020',
+        totalCost: 41.53,
+      },
+    ],
+  },
+  {
+    orderPlaced: 'July 4, 2020',
+    total: 17.44,
+    shipTo: 'JS Masher',
+    orderNumber: '114-2875557-9059409',
+    delivered: 'July 7, 2020',
+    item: [
+      {
+        type: 'adapter',
+        title: 'Gamecube controller adapter',
+        returnWindow: 'Aug 5, 2020',
+        totalCost: 15.98,
+      },
+    ],
+  },
+  {
+    orderPlaced: 'July 3, 2020',
+    total: 138.93,
+    shipTo: 'JS Masher',
+    orderNumber: '113-2883177-2648248',
+    delivered: 'Jun 5 2020',
+    item1: [
+      {
+        type: 'game',
+        title: 'Gamecube controller - Super Smash Bros',
+        returnWindow: 'Aug 4, 2020',
+        totalCost: 94.95,
+      },
+    ],
+    item2: [
+      {
+        type: 'book',
+        title: 'The Art of SQL',
+        returnWindow: 'Aug 4, 2020',
+        author: 'Faroult',
+        totalCost: 33.99,
+      },
+    ],
```

Comment:
this would be the only thing i would change. Instead of an `item` property on the for each order, and then just add another index within the `item` array when you have multiple items with the same order like this:

```
{
    orderPlaced: 'July 3, 2020',
    total: 138.93,
    shipTo: 'JS Masher',
    orderNumber: '113-2883177-2648248',
    delivered: 'Jun 5 2020',
    items: [
      {
        type: 'game',
        title: 'Gamecube controller - Super Smash Bros',
        returnWindow: 'Aug 4, 2020',
        totalCost: 94.95,
      },
      {
        type: 'book',
        title: 'The Art of SQL',
        returnWindow: 'Aug 4, 2020',
        author: 'Faroult',
        totalCost: 33.99,
      },
    ]
 }
 ```